node/inspector.ts

```diff
@@ -9,7 +9,435 @@ const messageCallbacksSymbol = Symbol("messageCallbacks");
+    | "error"
+    | "proxy"
+    | "promise"
+    | "typedarray"
+    | "arraybuffer"
+    | "dataview"
+    | "webassemblymemory"
+    | "wasmvalue";
+  export type WebDriverValueType =
+    | "undefined"
+    | "null"
+    | "string"
+    | "number"
+    | "boolean"
+    | "bigint"
+    | "regexp"
+    | "date"
+    | "symbol"
+    | "array"
+    | "object"
+    | "function"
+    | "map"
+    | "set"
+    | "weakmap"
+    | "weakset"
+    | "error"
+    | "proxy"
+    | "promise"
+    | "typedarray"
+    | "arraybuffer"
+    | "node"
+    | "window";
+  export type WebDriverValue = {
+    type: WebDriverValueType;
+    // Documentation calls it any
+    value?: unknown;
+    objectId?: string;
+  };
+  export type EntryPreview = {
+    key?: ObjectPreview;
+    value: ObjectPreview;
+  };
+  export type PropertyPreview = {
+    name: string;
+    type: ObjectType;
+    subtype?: ObjectSubtype;
+    value?: string;
+    valuePreview?: ObjectPreview;
+  };
+  export type ObjectPreview = {
+    type: ObjectType;
+    subtype?: ObjectSubtype;
+    description?: string;
+    overflow: boolean;
+    properties: PropertyPreview[];
+    entries: EntryPreview[];
+  };
+  export type CustomPreview = {
+    header: string;
+    bodyGetterId?: RemoteObjectId;
+  };
+  export type RemoteObjectId = string;
+  export type RemoteObject = {
+    type: ObjectType;
+    subtype?: ObjectSubtype;
+    className?: string;
+    // Documentation calls it any
+    value?: unknown;
+    unserializableValue?: UnserializableValue;
+    description?: string;
+    webDriverValue?: WebDriverValue;
+    objectId?: RemoteObjectId;
+    preview?: ObjectPreview;
+    customPreview?: CustomPreview;
+  };
+
+  type _RuntimeListenerMap = {
+    consoleAPICalled: (
+      type: CallType,
+      args: RemoteObject[],
+      executionContextId: ExecutionContextId,
+      timestamp: Timestamp,
+      stackTrace?: StackTrace,
+      context?: string,
+    ) => void;
+    exceptionRevoked: (reason: string, exceptionId: number) => void;
+    executionContextDestroyed: (
+      exectutionContextId: ExecutionContextId,
+    ) => void;
+    executionContextsCleared: () => void;
+    // !!! type on `hints` in the next line is an assumption
+    inspectRequested: (
+      object: RemoteObject,
+      hints: Record<string, unknown>,
+      executionContextId?: ExecutionContextId,
+    ) => void;
+    bindingCalled: (
+      name: string,
+      payload: string,
+      executionContextId: ExecutionContextId,
+    ) => void;
+  };
+  export type RuntimeListenerMap = PrefixKeys<"Runtime", _RuntimeListenerMap>;
+}
+
+// https://chromedevtools.github.io/devtools-protocol/v8/Schema/
+// By the empty type I mean that it was covered
+// type SchemaListenerMap = { };
+
+export type InspectorListenerMap =
+  & Console.ConsoleListenerMap
+  & Debugger.DebuggerListenerMap
+  & HeapProfiler.HeapProfilerListenerMap
+  & Profiler.ProfilerListenerMap
+  & Runtime.RuntimeListenerMap;
+// & SchemaListenerMap;
```

Comment:
This looks like it was unintentionally left in. Can you either remove it or add another comment explaining it.